src/main/modules/session/index.ts

```diff
@@ -22,7 +22,7 @@ export class SessionStorage {
         saveUninitialized: false,
         secret: config.get('session.secret'),
         cookie: {
-          httpOnly: false,
+          httpOnly: true,
```

Comment:
@tomxelliott - Looks like the respective test file needs to be fixed too

Reply:
Yep, am on it